hasmer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+WORKDIR /source
+
+COPY hasmer.sln hasmer.sln
+COPY hasmer-lsp/hasmer-lsp.csproj hasmer-lsp/hasmer-lsp.csproj
+COPY hasmer-cli/hasmer-cli.csproj hasmer-cli/hasmer-cli.csproj
+COPY libhasmer/libhasmer.csproj libhasmer/libhasmer.csproj
+RUN dotnet restore "hasmer.sln"
+
+COPY . .
+RUN dotnet publish -o /app
```

Comment:
Tbh I'm not 100% sure anymore why I don't use the same building commands as the ones from the `README.md`. I'll double-check and report back (or change the code).